src/Control/Carrier/FossaApiClient/Internal/FossaAPIV1.hs

```diff
@@ -277,7 +305,210 @@ instance ToDiagnostic FossaError where
         , indent 4 $ pretty . fpeUuid $ pfe
         , ""
         , reportDefectMsg
-        , "Please include Error UUID in your request."
+        , "Please include the error UUID in your request."
+        ]
+    InvalidUrlError url reason ->
+      vsep
+        [ "The URL provided is invalid."
+        , ""
+        , indent 4 $ "Provided:" <+> pretty url
+        , indent 6 $ "Reason:" <+> pretty reason
+        ]
+    StatusCodeError ereq eres ->
+      vsep
+        [ "The FOSSA service returned an unexpected status code:"
```

Comment:
[optional] [prefer] fossa endpoint over fossa service / fossa backend. ame carries for App.Support